### DISCUSSION TITLE
Nombre del proyecto

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#15] [Ideas]
Deberíamos darle un nombre al proyecto? Así como existe "MediaWiki", como existe "Wordpress", etc, se le ocurre algún nombre para darle al sistema y usarlo por ejemplo en el documento de especificaciones, en el copyright, etc?

**Comment by AhumadaCeleste** [comment]
Comparto palabras como sugerencia para armarlo. Remate o subasta en ingles= auction/ puja en ingles=  bid
o algo en nuestra lengua, Pujaremate.com

**Comment by NataliaAlvarezIspc** [comment]
Podemos darle un nombre sencillo al proyecto, como "Página de Subastas" y luego pensar un nombre específico para el  copyright como Rematequeremate.com

**Comment by Brian-Almada** [comment]
bidOn.com.ar
tiro esa.. hasta que se me ocurra otra!

**Comment by claubott** [comment]
Esto no le gana al bidOn, pero encontré por un sinónimo de subasta:
**almoneda** (sust. fem.)
1. Subasta de bienes, generalmente a bajo precio.
"traficaba habilidosamente en almonedas y compraventas, antiguallas e imitaciones"
2. Establecimiento donde se realiza este tipo de venta pública.

**Comment by rpgrca** [comment]
Está difícil, me gustan las dos, bidon.com.ar se puede registrar, no así bidon.ar que alguien ya registró. O incluso podríamos registrar almoned.ar que sería el verbo de almoneda. No es necesario registrar el sitio pero supongo que nos daría unos puntitos extras.

**Comment by NataliaAlvarezIspc** [comment]
Resolvimos por votación que el nombre bidOn.com quede como nombre para la página de subastas y Almoneda quede como nombre de grupo (o empresa). Damos así por terminado este tema. 
Ahora debemos diseñar un  logo para Almoneda y retocar el logo de bidOn.
